Dockerfile

```diff
@@ -13,25 +15,21 @@ RUN set -ex; \
       x11vnc \
       xterm \
       python3-cryptography python3-PyJWT \
-      xvfb-run dosbox unzip
+      xvfb-run dosbox unzip && \
+    zypper --non-interactive clean && \
+    rm -f /etc/zypp/credentials.d/SCCcredentials
```

Comment:
it fails to build on SLES: /etc/zypp/credentials.d/SCCcredentials is a tmpfs mount point (when using podman)

Reply:
Whoops, that was left over from a previous try when injecting the credentials into the image differently. I have removed this line.